.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,52 @@
+name: UnitTests
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+
+jobs:   
+  run-unit-tests:
+
+    runs-on: ubuntu-latest
+    container: compucorp/civicrm-buildkit:1.0.0
+    
+    env:
+      REPOSITORY_NAME: ${{ github.event.repository.name }}
+      CIVICRM_CORE: 5.24.6
+      CIVICRM_EXTENSIONS_DIR: /buildkit/civicrm-buildkit/build/drupal-clean/web/sites/all/modules/civicrm/tools/extensions
+    
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Config mysql database as per CiviCRM requirement 
+        run: echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" | mysql -u root --password=root --host=mysql
+
+      - name: Config amp
+        run : amp config:set --mysql_dsn=mysql://root:root@mysql:3306
+      
+      - name: Build Drupal site 
+        run: civibuild create drupal-clean --civi-ver ${CIVICRM_CORE} 
```

Comment:
Also I was wondering about this. There might be a change that a test will pass on our local environment but fail when the workflow is ran. 
It's possible that the test depends on a patch that has already been applied to the civicrm on compuclient and the patch is for a future version of Civicrm and so the test will fail.
What do we do in that case?

Reply:
@tunbola  yes, the civicrm version must be the same with compuclient version. if civicrm core is a patch version in compuclient, we will need to patch to this workflow too.

Reply:
The process of patching the workflow to match the civicrm version in compuclient, is it an automated process? @erawat 
Or we need to do something manual?

Reply:
@tunbola  it should be automated.